configure.ac

```diff
@@ -142,6 +142,19 @@ AM_COND_IF([HAVE_MKINITCPIO],
            ])
 AC_CONFIG_FILES([dist/initcpio/install/tpm2-totp dist/initcpio/install/plymouth-tpm2-totp])
 
+AC_ARG_WITH([udevdir],
+            AS_HELP_STRING([--with-udevdir=DIR], [udev directory]),,
+            [PKG_CHECK_VAR([with_udevdir], [udev], [udevdir],, [with_udevdir="$libdir/udev"])])
+AC_SUBST([UDEVDIR], [$with_udevdir])
+
+AC_ARG_WITH([systemdsystemunitdir],
+            AS_HELP_STRING([--with-systemdsystemunit=DIR], [systemd system unit directory]),,
+            [PKG_CHECK_VAR([with_systemdsystemunitdir], [systemd], [systemdsystemunitdir],,
+                           [with_systemdsystemunitdir="$libdir/systemd/system"])])
+AC_SUBST([systemdsystemunitdir], [$with_systemdsystemunitdir])
+
+AC_CONFIG_FILES([dist/initcpio/install/sd-tpm2-totp dist/tpm2-totp.service dist/tpm2-totp.timer])
```

Comment:
`tpm2-totp.timer` doesn't need to be in `AC_CONFIG_FILES` since it doesn't have any variable substitutions. Instead you could rename `tpm2-totp.timer.in` to just `tpm2-totp.timer` and add it to `EXTRA_DIST` here https://github.com/tpm2-software/tpm2-totp/blob/4efa23d201cd6aff7c13004eeaae1c54d40f6702/Makefile.am#L211
so that it gets included into the release tarball.

Just a minor comment though, I don't really mind having it in `AC_CONFIG_FILES` either.

Reply:
My rationale was to leave it as a template file, as one might want to parameterize the paths to the binaries in the future (e.g. substitute `/usr/bin` as `@BINDIR@`).
I had looked in `configure.ac` if there was any substitution defined to abstract away `$PATH` or `/usr/bin`, but as there wasn't any, I simply relied on systemd having a sensible default `$PATH` and called the binaries without an absolute path.

Technically, also `tpm2-totp.service.in` doesn't rely on parameter substitution any more, as it no longer depends on `show-tpm2-totp` and so no longer contains any references to `@HELPERSDIR@`.

If you prefer non-template files for this merge, just let me know.

Reply:
I don't really mind, the files can stay in `AC_CONFIG_FILES` as-is.